### PR TITLE
Make the code Prettier

### PR DESCRIPTION
**Pull request by fatfisz**
I used those rules:
* `printWidth: 100` because 80 is sometimes too small
* `semi: false` to preserve what the standard StandardJS behavior (maybe we could live with semis, though?)
* `singleQuote: true` to preserve single-quoted strings as a default (double quotes are used only when single quotes are escaped inside)
* `trailingComma: 'all'` for diff sanity

Btw. I added stricter rules for `'use strict'`, as it's unnecessary in ES modules (they are strict by default). The build output reflects that, as `'use strict'` is still generated automatically where needed.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    sourceType: 'script',
+  },
+  parser: 'babel-eslint',
+
+  rules: {
+    strict: [2, 'global'],
+    'prettier/prettier': [
+      2,
+      {
+        printWidth: 100,
```
Comment (declandewet) [inline .eslintrc.js]: I'd actually prefer this to be 80. I use 80 as my limit in all my projects as I work predominantly on a single tiny monitor.

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        semi: false,
+        singleQuote: true,
```
Comment (declandewet) [inline .eslintrc.js]: Let's enforce semicolons from now on.

File: contributing.md [hunk 3]
```diff
@@ -11,5 +11,6 @@
 
 Please include _passing_ tests for any new logic or features you include in your contribution. This project uses [AVA](/sindresorhus/ava) to run tests and is setup to look for files that end with `.test.js` in the `src` directory.
 
-Furthermore, this project uses the [StandardJS](/feross/standard) code style,
-and tests will _fail_ if your code does not adhere to this style.
+Furthermore, this project uses [Prettier](/prettier/prettier) for code formatting,
```
Comment (declandewet) [inline contributing.md]: This doesn't make sense as Prettier is a code formatter, not a linter. It should be "Prettier and ESLint"

File: package.json [hunk 4]
```diff
@@ -103,19 +105,13 @@
     "codecov": "npm run coverage && codecov",
     "coverage": "nyc report --reporter=lcov",
     "doctoc": "doctoc readme.md --title \"\n## Table of Contents\"",
-    "lint": "standard | snazzy",
+    "lint": "eslint .*rc.js src/**/*.js",
+    "lint:fix": "eslint --fix .*rc.js src/**/*.js",
```
Comment (declandewet) [inline package.json]: Does this automatically run prettier as well? When I was reading about it I was under the impression that it didn't and you had to call `prettier` CLI yourself before `eslint`...

Comment (fatfisz) [reply package.json]: Yup, it does. By using a glue plugin for ESLint we make it run Prettier and all the formatting errors are normal ESLint errors. And the rule `prettier/prettier` is fixable, which just means that Prettier is ran on the code 🙂

**Comment by fatfisz** [comment]
Tomorrow I'll move the config to `.prettierrc` then; the ESLint plugin works with that as well. I'll also apply other requested changes.

**Comment by fatfisz** [comment]
I had a few spare minutes, so I updated everything as requested 🙂

**Comment by declandewet** [comment]
Thanks @fatfisz - you're a star 🌟 There seems to be some issues with CI so I'll re-run those and then we can get this merged :)

**Pull request merged**